lib/modules/datasource/github-runners/readme.md

```diff
@@ -1,11 +1,12 @@
-This datasource returns a list of all _stable_ runners that are hosted by GitHub.
-This datasource ignores beta releases.
+This datasource returns a list of all runners that are hosted by GitHub.
 The datasource is based on [GitHub's `runner-images` repository](https://github.com/actions/runner-images).
 
 Examples: `windows-2019` / `ubuntu-22.04` / `macos-13`
```

Comment:
```suggestion
Examples: `windows-2022` / `ubuntu-22.04` / `macos-13`
```

2022 is the latest stable version for Windows, according to the upstream readme. So I'm updating this reference in our docs.

Ubuntu 24.04 will become latest soon, but it'll update our docs myself then. 😄